src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java

```diff
@@ -1,22 +1,32 @@
 package com.commercetools.sync.commons.utils;
 
 import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.client.RetrySphereClientDecorator;
+import io.sphere.sdk.client.ServerErrorException;
 import io.sphere.sdk.client.SphereAccessTokenSupplier;
 import io.sphere.sdk.client.SphereClient;
 import io.sphere.sdk.client.SphereClientConfig;
 import io.sphere.sdk.http.AsyncHttpClientAdapter;
 import io.sphere.sdk.http.HttpClient;
+import io.sphere.sdk.retry.RetryContext;
+import io.sphere.sdk.retry.RetryRule;
 import org.asynchttpclient.AsyncHttpClient;
 import org.asynchttpclient.DefaultAsyncHttpClient;
 import org.asynchttpclient.DefaultAsyncHttpClientConfig;
 
 import javax.annotation.Nonnull;
+import java.time.Duration;
 import java.util.concurrent.TimeUnit;
 
+import static io.sphere.sdk.retry.RetryAction.ofScheduledRetry;
+import static io.sphere.sdk.retry.RetryPredicate.ofMatchingErrors;
+import static java.util.Collections.singletonList;
+
 public class ClientConfigurationUtils {
     private static BlockingSphereClient ctpClient;
     private static final long DEFAULT_TIMEOUT = 30;
     private static final TimeUnit DEFAULT_TIMEOUT_TIME_UNIT = TimeUnit.SECONDS;
+    private static final int RETRY_ON_5XX_ATTEMPTS_LIMIT = 27;
```

Comment:
Also why is it a global variable and not passed as a parameter to the method it's used in?